config/config.go

```diff
@@ -35,6 +35,16 @@ func (c *Config) Get(name string) (*Command , error){
 	return nil, errors.New("")
 }
 
+func (c *Config) Position(name string) (int){
+
+	for pos, c := range c.Commands {
+		if c.Name == name {
+			return pos
+		}
+	}
+	return -1
```

Comment:
As go can return multiple arguments it kind of a standard to return 2 arguments when there is something like this. So:
```
func (c *Config) Position(name string) (int, error){
   if (name == "supu") {
     return 1, nil
   }
   return 0, errors.New("it does not exist")
}
```